server/app/views/admin/programs/ProgramImageView.java

```diff
@@ -273,15 +272,12 @@ private DivTag createImageUploadForm(
             .isDisabled());
     buttonsDiv.with(deleteButton);
 
-    // TODO(#5676): Replace with final UX once we have it.
     return div()
         .withClass("mt-10")
         .with(fullForm)
         .with(buttonsDiv)
         .with(p("Note: Image description is required before uploading an image.").withClass("mt-1"))
         .with(fileUploadViewStrategy.footerTags());
-
-    // TODO(#5676): Warn admins of recommended image size and dimensions.
```

Comment:
Warning is included in the text above the file upload -- see 
![image](https://github.com/civiform/civiform/assets/147177873/ddaaa889-226d-4e00-bb5a-3bca08debd98)